src/main/java/randoop/main/Main.java

```diff
@@ -80,10 +81,15 @@ public void nonStaticMain(String[] args) {
     } catch (BugInRandoopException e) {
       System.out.println();
       System.out.println("Randoop failed in an unexpected way.");
-      System.out.println("Please report at https://github.com/randoop/randoop/issues .");
-      e.printStackTrace();
-      System.err.flush();
-      success = false;
+      System.out.println("Please report at https://github.com/randoop/randoop/issues.");
```

Comment:
The space before the period was intentional, so that people can click on the URL without their tools interpreting the "." as part of the URL.